.vitepress/theme/components/Integrations.vue

```diff
@@ -29,7 +29,7 @@ const items = [
   },
   {
     title: 'Svelte',
-    link: '/guide/integrations/svelte-sapper',
+    link: '/guide/integrations/svelte-preprocess',
```

Comment:
I think just `svelte` will be good, WDYT.

Reply:
i did this difference because, we also have svelteKit, which will include two sections.
Svelte could be setup with either svelte-windicss-preprocess or with the bundler plugin for the used bundler, do not know how to make difference. WDYT?

Reply:
Got it.